src/renderers/native/ReactNativeViewConfigRegistry.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactNativeViewConfigRegistry
+ * @flow
+ */
+
+'use strict';
+
+type ReactNativeBaseComponentViewConfig = {
+  validAttributes: Object,
+  uiViewClassName: string,
+  propTypes?: Object,
+};
+
+const viewConfigs = new Map();
+
+const prefix = 'topsecret-';
```

Comment:
I guess this will bloat devtools a little bit if all host components are prefixed with this?